src/marshmallow/class_registry.py

```diff
@@ -11,16 +11,19 @@
 from marshmallow.exceptions import RegistryError
 
 if typing.TYPE_CHECKING:
-    from marshmallow import Schema
+    # noinspection PyUnresolvedReferences
+    from marshmallow import Schema  # noqa: F401
```

Comment:
linters 'think' that `Schema` is unused because I created a separate `SchemaType` annotation variable, so `Schema` is not used to annotate variables